projects/mlir-wheel/setup.py

```diff
@@ -0,0 +1,15 @@
+#  Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+#  See https://llvm.org/LICENSE.txt for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+import os
+
+from setuptools import setup
+
+
+version = os.environ.get("WHEEL_VERSION", "0.0.0+DEADBEEF").replace("_", "+")
```

Comment:
Just to double check: is it correct to replace each `_` with a `+`?

Reply:
in `WHEEL_VERSION` yes (python version strings can't have `_` in them).